### PR TITLE
Change default value of arrowstyle for undirected graphs

### PR DESCRIPTION
**Pull request by dtekinoglu**
I changed default arrowstyle to "-" for undirected graphs. A person can still set another arrowstyle if he/she wants. But if `arrows=True` for undirected graphs without specifying an arrowstyle, no arrow-heads will be shown as expected.

### CODE REVIEW

File: networkx/drawing/nx_pylab.py [hunk 1]
```diff
@@ -678,6 +681,13 @@ def draw_networkx_edges(
     # undirected graphs (for performance reasons) and use FancyArrowPatches
     # for directed graphs.
     # The `arrows` keyword can be used to override the default behavior
+
+    if not G.is_directed() and arrowstyle == None:
+        arrowstyle = "-"
+
+    if G.is_directed() and arrowstyle == None:
+        arrowstyle = "-|>"
```
Comment (jarrodmillman) [inline networkx/drawing/nx_pylab.py:689]:
```suggestion
    if arrowstyle == None:
        if G.is_directed():
            arrowstyle = "-|>"
        else:
            arrowstyle = "-"
```

Comment (dtekinoglu) [reply networkx/drawing/nx_pylab.py]: Thank you @jarrodmillman , I committed the suggestion.

**Review by dschult — APPROVED** [review]
I think this default makes more sense. The default on `arrows` should make this hard to run into, but it is good to have the code explicit and the documentation say what is being done.
I approve this PR.

**Comment by rossbar** [comment]
> I think this default makes more sense. The default on arrows should make this hard to run into, but it is good to have the code explicit and the documentation say what is being done.

I'm actually not sure about this one. The behavior that this PR switched to is now `arrows=True` results in *no* arrowheads for undirected graphs. I agree that the previous behavior for undirected graphs (one-directional arrowheads) didn't make sense, but I don't know that this is quite right either. In the [accompanying discussion](https://github.com/networkx/networkx/issues/5466#issuecomment-1087982884) it was suggested to switch to `<->` for undirected graphs when `arrows=True` - I think that's a better approach than what's taken here.

I'd vote to update this along with some tests to make sure we're covering all the input combinations (directed/undirected graphs with arrowstyle=None and arrows=True/False). Thoughts?

**Pull request merged**